apps/self.id/src/client/env.ts

```diff
@@ -22,14 +25,13 @@ export async function authenticate(
   address: string
 ): Promise<SelfID> {
   const authProvider = new EthereumAuthProvider(provider, address)
-  const did = await client.authenticate(authProvider)
-  client.ceramic.did = did
+  const did = await client.authenticate(authProvider, true)
```

Comment:
What does true mean here?

Reply:
I attaches the did to ceramic, basically what the removed line used to do.

Reply:
ok, why wouldn't it be attached by default?